hack/dockerfiles/test.Dockerfile

```diff
@@ -47,20 +47,20 @@ COPY --from=runc /usr/bin/runc /usr/bin/runc
 COPY --from=containerd /go/src/github.com/containerd/containerd/bin/containerd* /usr/bin/
 
 
-FROM buildkit-base AS buildd-standalone
+FROM buildkit-base AS buildd.oci_only
 ENV CGO_ENABLED=0
-RUN go build -ldflags '-d'  -o /usr/bin/buildd-standalone -tags standalone ./cmd/buildd
+RUN go build -ldflags '-d'  -o /usr/bin/buildd.oci_only -tags no_containerd_worker ./cmd/buildd
 
-FROM buildkit-base AS buildd-containerd
+FROM buildkit-base AS buildd.containerd_only
 ENV CGO_ENABLED=0
-RUN go build -ldflags '-d'  -o /usr/bin/buildd-containerd -tags containerd ./cmd/buildd
+RUN go build -ldflags '-d'  -o /usr/bin/buildd.containerd_only -tags no_oci_worker ./cmd/buildd
 
 FROM registry:$REGISTRY_VERSION AS registry
 
 FROM unit-tests AS integration-tests
 COPY --from=buildctl /usr/bin/buildctl /usr/bin/
-COPY --from=buildd-containerd /usr/bin/buildd-containerd /usr/bin
-COPY --from=buildd-standalone /usr/bin/buildd-standalone /usr/bin
+COPY --from=buildd.containerd_only /usr/bin/buildd.containerd_only /usr/bin
```

Comment:
Are these binaries still required here?

Reply:
I only meant that if tests use `buildd` then `integration-tests` stage shouldn't need them. I didn't mean to imply that `buildd.oci_only` makes more sense than `buildd.containerd_only`.

If you think that  `buildd.containerd_only` is unnecessary for some other reason we could still try to build it. But it can be in `hack/test` then rather than in Dockerfile (and that can be follow up).

Reply:
ok, updated